demo/cms/components/decomButton.jsx

```diff
@@ -0,0 +1,10 @@
+const React = require('react');
+
+const DecomButton = props => (
+	<div className="decommission-override">
+		Stand-in decom component for example app
+		{props.entireRecord.lifecycleStage}
```

Comment:
```suggestion
		{props.entireRecord.someString}
```

As you'll be working with the example schema even though you can't access lifecycleStage, you can use other properties defined on MainType to test that the values are getting through

Reply:
Ah what I meant was, I can't even access .decommission-override. 
I've just pushed up a failing test to show what I mean. 
`cy.get('.decommission-override').should('exist'); // fails`

Reply:
That's what I meant about the .yaml file. I think that dictates what renders on the cypress version, but divs which aren't inside a named component also render out, which is how I get access to .additional-edit-hydration-container
I can test that that div exists but that's it, unless there's a way I can't see to get other things to render.